.ci/packaging.groovy

```diff
@@ -35,56 +36,49 @@ pipeline {
       }
     }
     stage('Linux'){
-      agent { label 'ubuntu && immutable' }
       options { skipDefaultCheckout() }
-      when {
-        beforeAgent true
-        expression { return params.linux }
-      }
-      environment {
-        SNAPSHOT = "true"
-        PLATFORMS = "+linux/armv7 +linux/ppc64le +linux/s390x +linux/mips64"
-        HOME = "${env.WORKSPACE}"
-      }
-      steps {
-        deleteDir()
-        unstash 'source'
-        dir("${BASE_DIR}"){
-          sh(label: 'Release', script: './dev-tools/jenkins_release.sh')
+      matrix {
+        agent { label 'ubuntu && immutable' }
+        axes {
+          axis {
+            name 'PLATFORMS'
+            values '+linux/armv7', '+linux/ppc64le', '+linux/s390x', '+linux/mips64', 'darwin'
```

Comment:
Maybe I'm missing something, but don't we build windows packages?

Reply:
not from the betas-ci, we only have this jobs https://beats-ci.elastic.co/job/elastic+beats+master+package/
I've check https://github.com/elastic/beats/blob/master/dev-tools/mage/platforms.go#L29-L70 and there is a bunch of platforms we do not build on the beats-ci, Should we build them?

Reply:
checking the logs, I think does no matter what you put on PLATFORMS env var you always build all the packages

Reply:
yep, checking the output of one of the old jobs, the PLATFORMS env var it is ignored always build all the packages